### DISCUSSION TITLE
Need Contributers for my project || Open Source || MERN Stack Chat Application

### DISCUSSION THREAD

**Discussion by Muntajir11** [community/community#139613] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Show & Tell

### Body

If anyone is interested in working with me on my project. This project is completed just need to add few features!!
 ᑕOᑎᑎEᑕT 🪢 - A real-time messaging application designed for seamless and instant communication. Built with the MERN stack (MongoDB, Express, React, Node.js) and Socket.IO for real-time chat functionality, ᑕOᑎᑎEᑕT 🪢 ensures your conversations are swift and secure.

You can also support me by starring my repository if you like it: https://github.com/Muntajir11/RealTimeChat

I would greatly appreciate any feedback you can give me.

**Comment by LiteBrite82** [comment, marked as answer]
Hi @Muntajir11 , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by Muntajir11** [reply]
Ok I didn't know that. Thanks
